utilities/update_docs_version.sh

```diff
@@ -17,6 +17,9 @@ if [ "${RELEASED_VERSION##*-}" != "SNAPSHOT" ]; then
         sed -ri "s/:[0-9]+\.[0-9]+\.[0-9]+'/:${RELEASED_VERSION}'/g" ${item}/README.md
         sed -ri "s/\"[0-9]+\.[0-9]+\.[0-9]+\"/\"${RELEASED_VERSION}\"/g" ${item}/README.md
     done
+
+    echo "Changing version to $((RELEASED_VERSION + 1))-SNAPSHOT in gcloud-java-nio-example README"
```

Comment:
Actually, I see that a user might clone the repo and try this for him/herself, in which case the version will have a SNAPSHOT in it; I'll post commands for that.